### PR TITLE
Updated French Translation

### PR DESCRIPTION
**Pull request by D4rKiller342**
Made some updates to the French translation:
what I changed
puzzleDlcViewNow from "View Dlc" --> "Voir le Dlc"

restart from "Relancer" --> "Reco"

puzzleReportComplete:
	title: from "Merci pour votre retour!" --> "Merci pour votre commentaires!"

puzzleEditorSettings:
	clearBuildings: from "Clear Buildings" --> "Retirer les bâtiments"
	resetPuzzle: from "Reset Puzzle" --> "Réinitialiser le puzzle"

puzzleEditorControls:
	instructions:
		-4. from "Click the <strong>lock button</strong> on a building to disable it."
		 --> "Cliquez sur le <strong>bouton verrouillage</strong> sur un bâtiment pour le désactiver."

		-5. from "Once you click review, your puzzle will be validated and you can publish it."
		 --> "Une fois que vous aurez cliqué sur "réviser", votre puzzle sera validé et vous pourrez le publier."

		-6. from "Upon release, <strong>all buildings will be removed</strong> except for the Producers and Goal Acceptors - That's the part that the player is supposed to figure out for themselves, after all :)" 
		--> "A la publication, <strong>tous les bâtiments seront retirés</strong> excepté les Producteurs et les Récépteur d'Objectif - C'est la partie que le joueur est censé découvrir par lui-même, après tout :)"

puzzleMetadata:
	shortKey: from "Short Key" --> "Code"
	
shapeTooltipAlwaysOn: 
	title: from "Shape Tooltip - Show Always" --> "Toujours afficher - l'Info-bulle de la forme"
	description: from "Whether to always show the shape tooltip when hovering buildings,
                instead of having to hold 'ALT'." --> "Si vous souhaitez toujours afficher l'infobulle de la forme lors du survol des bâtiments,
                au lieu de devoir maintenir la touche 'ALT' enfoncée."
	
mappings:
	showShapeTooltip: from "Show shape output tooltip" --> "Afficher l'infobulle pour la sortie de la forme"

difficulties:
	easy: from "Easy" --> "Facile"
	medium: from "Medium" --> "Medium"
	hard: from "Hard" --> "Difficile"
	unknown: from "Unrated" --> "Non évalué"
dlcHint: from "Purchased the DLC already? Make sure it is activated by right clicking
	shapez.io in your library, selecting Properties > DLCs." --> "Vous avez déjà acheté le DLC ? Assurez-vous qu'il est activé en faisant un clic droit sur
        shapez.io dans votre bibliothèque, puis sélectionnez Propriétés > DLC."
search:
	action: from "Search" --> "Recherchez"
	placeholder: from "Enter a puzzle or author name" --> "Entrez un puzzle ou un nom d'auteur"
	includeCompleted: from "Include Completed" --> "Inclusion compléter"
	difficulties:
		any: from "Any Difficulty" --> "N'importe quelle difficulté"
		easy: from "Easy" --> "Facile"
		medium: from "Medium" --> "Medium"
		hard: from "Hard" --> "Difficile"
	durations:
		any: from "Any Duration" --> "Toute durée"
		short: from "Short (< 2 min)" --> "Court (< 2 min)"
		medium: from "Normal" --> "Normale"
backendErrors:
	    too-many-likes-already: from "The puzzle already got too many likes. If you still want
        to remove it, please contact support@shapez.io!" --> "Ce puzzle a déjà reçu trop de likes. Si vous voulez toujours le supprimer, veuillez contacter support@shapez.io !"
    no-permission: from "You do not have the permission to perform this action." --> "Vous n'avez pas la permission d'effectuer cette action."

### CODE REVIEW

**Comment by EmeraldBlock** [comment]
there's no need to copy down everything you changed, the ["Files changed"](https://github.com/tobspr/shapez.io/pull/1324/files) tab shows that

**Comment by daanbreur** [comment]
Yeh. Also formatting is a thing.

File: translations/base-fr.yaml [hunk 1]
```diff
@@ -249,7 +249,7 @@ dialogs:
             unsolvable: Irrésolvable
             trolling: Troll
     puzzleReportComplete:
-        title: Merci pour votre retour!
+        title: Merci pour votre commentaires!
```
Comment (libetl) [inline translations/base-fr.yaml:252]:
```suggestion
        title: Merci pour votre commentaire !
```

**Comment by tobspr** [comment]
Can you check the conflicts?

**Pull request closed without merging**